bescort.lic

```diff
@@ -680,7 +683,15 @@ class Bescort
       focus_shard(shard) if DRStats.circle < 100
       3.times { harness_mana(20) }
       waitcastrt?
-      DRStats.circle > 99 ? cast?('cast grazhir') : cast?("cast #{shard}")
+      if DRStats.circle > 99
+        unless cast?('cast grazhir')
+          echo "You may need to move to another room"
+          fput('release mana')
+          exit
+        end
+      else
+        cast?("cast #{shard}")
```

Comment:
No return checked here because this is casting in known rooms (where shards are).